cartographer/mapping/internal/2d/pose_graph_2d.cc

```diff
@@ -1240,12 +1240,31 @@ void PoseGraph2D::TrimmingHandle::TrimSubmap(const SubmapId& submap_id) {
   // Remove all 'data_.constraints' related to 'nodes_to_remove'.
   {
     std::vector<Constraint> constraints;
+    std::set<SubmapId> unconstrained_submap_ids;
```

Comment:
Took me a while to understand this. Can we add a comment? This is the set of other submaps to which the `nodes_to_remove` had inter-submap constraints before and which lost at least some of their constraints, right? Maybe renaming is sufficient. My attempt: `other_submap_ids_losing_some_constraints`. What do you think?

Reply:
Changed, I removed "some" as it doesn't add much value IMO.